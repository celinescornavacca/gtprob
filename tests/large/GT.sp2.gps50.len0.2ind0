((((A3,(((A24,(A43,A44)),A21),((A8,(A22,A23)),A6))),((A4,A5),((A17,(A41,A42)),((((A29,A30),A18),A9),(A13,A14))))),(((((((A48,A49),A47),(((A31,A32),A11),(((A20,(A45,A46)),A7),(A25,A26)))),(A2,((A37,A38),(A27,A28)))),(A12,(A15,(A33,A34)))),A0),(A1,(((A19,(A39,A40)),A16),((A35,A36),A10))))),(((((((B10,B11),(B18,((B44,B45),B29))),((B48,B49),((B38,B39),(B19,B20)))),B0),(B2,((B8,(B46,B47)),(((B6,B7),(B26,B27)),B4)))),((((B23,B24),(B17,(B30,B31))),(B1,(B14,B15))),((B33,B34),B9))),(((B35,B36),(B5,(B12,B13))),((B16,((B28,((B37,(B42,B43)),B32)),B25)),((B21,B22),(B3,(B40,B41)))))));
