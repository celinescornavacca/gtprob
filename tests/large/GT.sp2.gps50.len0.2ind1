(((((((A9,A10),A1),(((A42,A43),(((A40,A41),A21),(A24,(A48,A49)))),(A13,A14))),((A26,A27),(A4,((A28,A29),A23)))),A0),(((A32,A33),(A44,A45)),((((A3,(((A25,(A46,A47)),(A19,((((A30,A31),A22),(A34,A35)),A20))),(A38,A39))),(((A11,(A17,A18)),A8),A7)),(((A15,A16),A2),(A5,A6))),((A36,A37),A12)))),((B1,((((B19,B20),(B3,B4)),(((B8,(B32,B33)),B7),(B48,B49))),((B36,B37),(B16,(B22,(B28,B29)))))),((((B44,B45),((B17,B18),B13)),B2),(B0,((((((B40,B41),(B23,B24)),B21),B10),((((B30,B31),(B26,B27)),B9),B6)),((B12,(B14,(((B35,(B42,B43)),B34),B15))),((B11,((((B46,B47),B39),B38),B25)),B5)))))));
