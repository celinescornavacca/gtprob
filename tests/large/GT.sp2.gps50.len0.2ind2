((((((A18,A19),((((A32,A33),A20),A16),A12)),(A6,(((A48,A49),A22),(A46,A47)))),((A7,A8),(A36,A37))),(((((((A13,(A14,A15)),(A42,A43)),((A25,A26),A9)),((A0,((A3,A4),((A44,A45),A21))),(A1,A2))),(A5,((A34,A35),((A40,A41),((A38,A39),A29))))),(A17,((A23,A24),(A30,A31)))),((A27,A28),(A10,A11)))),(((B1,(((B23,B24),(B16,B17)),B2)),((((B5,B6),((B34,B35),(((B21,B22),((B36,B37),((B40,B41),B12))),(B15,(B48,B49))))),(((((B25,B26),B18),B14),B11),(((B27,B28),(B19,((B44,B45),(B38,B39)))),(B13,(B20,((B46,B47),((B32,B33),B29))))))),B0)),(((((B30,B31),((B9,B10),B8)),B4),B3),((B42,B43),B7))));
