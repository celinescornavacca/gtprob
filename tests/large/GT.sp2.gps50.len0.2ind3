(((((((A14,A15),A2),(((A16,(A40,A41)),(A29,(A38,A39))),A1)),(((A7,(A8,((A23,A24),(A21,(A36,A37))))),((A22,(A25,A26)),A20)),((A11,A12),A4))),(((A3,(A6,(A27,A28))),((((A48,A49),A35),(((A44,A45),(A46,A47)),A5)),(((A10,(A13,(A32,(A42,A43)))),((A17,A18),(A33,A34))),A9))),((A30,A31),A19))),A0),(((B0,(((((B36,B37),((B38,B39),B29)),(B27,B28)),(B20,B21)),B1)),((B4,((B11,(B44,B45)),(B32,B33))),((((B34,B35),((B25,(B30,B31)),(B12,B13))),B3),(B6,B7)))),((((B42,B43),(B5,((B14,B15),(B18,B19)))),((B8,B9),(B2,((B48,B49),(B16,B17))))),((((B46,B47),B10),(B22,B23)),(B24,((B40,B41),B26))))));
