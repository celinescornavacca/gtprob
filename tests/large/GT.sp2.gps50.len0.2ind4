(((((((A16,((A31,A32),A19)),A2),A0),((((A41,(A48,A49)),((A39,A40),(A21,A22))),(A7,(A10,(A37,A38)))),(((A13,(A14,A15)),((A11,((A35,A36),(A44,A45))),A5)),(A3,A4)))),(A42,A43)),((((A8,A9),((A20,(A29,A30)),((A18,(A46,A47)),A12))),(((((A23,A24),A17),A6),(A27,A28)),(((A33,A34),A26),A25))),A1)),((((B16,(B31,(B35,B36))),(((B8,(B26,(B29,(B37,B38)))),(B11,B12)),(B4,B5))),((B14,B15),B9)),((((((B48,B49),B34),B27),((B13,(B17,((B44,B45),B30))),(B1,((((B46,B47),B28),B6),((B18,(B32,B33)),(B10,(B20,B21))))))),(((B22,B23),(B39,B40)),(B0,(B19,(B24,B25))))),((B7,(B41,(B42,B43))),(B2,B3)))));
