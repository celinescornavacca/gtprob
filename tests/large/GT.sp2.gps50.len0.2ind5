((((((((A39,A40),A30),((A23,A24),A18)),(A41,A42)),(A1,(A32,(A35,A36)))),((A9,(A25,(A31,(A43,A44)))),((A7,(A14,A15)),((((A8,(((A48,A49),A45),((A12,A13),(A26,A27)))),A5),A2),((A3,((A16,A17),(A22,((A37,A38),(A46,A47))))),(((A20,A21),A19),A4)))))),((A10,A11),(A0,(A6,((A33,A34),(A28,A29)))))),((((B3,B4),((((B36,B37),(B29,B30)),B7),((((B42,B43),(B27,B28)),(((B18,B19),((B20,B21),B17)),((B44,B45),B32))),((B38,B39),(B15,B16))))),((B12,B13),B1)),(((((B2,((B6,(B11,((B33,(B48,B49)),B22))),(B14,(B23,B24)))),(((B46,B47),((B40,B41),B10)),B8)),B0),(B5,((B25,B26),B9))),(B31,(B34,B35)))));
