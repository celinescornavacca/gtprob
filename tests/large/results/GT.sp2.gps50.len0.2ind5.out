-349.760418609
